index.htm

```diff
@@ -22,14 +22,32 @@
       <li><a href="./sites/begonia-details.htm">Shop</a></li>
     </ul>
     <input type="search" id="search-box" name="search" placeholder="Search...">
-    <a href="#" class="button outlined-button">Login</a>
+    <a href="#" class="button outlined-green-button">Login</a>
```

Comment:
its more maintainable when you name that colors: primary, secondary. If you need to change it from green to red then, you just need to change the actual colors in the css not all the names. 

The classes could potentially used in millions of htmls, the css just once.

Reply:
Refactoring complete.